### PR TITLE
Refactor conversion of quad to Date on timeline in QuadConverter

### PR DESCRIPTION
**Pull request by kevinrobinson**
Hi @erose!  In fixing up https://github.com/studentinsights/studentinsights/pull/119, I noticed there was some date conversion that we could pull into the `QuadConverter` helper object, since it's small and generic.  I did that here and then added some quick specs.

### CODE REVIEW

File: app/assets/javascripts/student_profile_v2/page_container.js [hunk 1]
```diff
@@ -91,10 +91,6 @@
       return [nowMoment.clone().subtract(1, 'year').toDate(), nowMoment.toDate()];
     },
 
-    cumulativeCountQuads: function(attendanceEvents) {
```
Comment (kevinrobinson) [inline app/assets/javascripts/student_profile_v2/page_container.js]: Unrelated, just noticed this was unused.

**Comment by kevinrobinson** [comment]
@erose this seem like a good change?

**Comment by erose** [comment]
I APPROVE!

**Comment by kevinrobinson** [comment]
:+1:

**Pull request merged**